src/A1_csv_to_json.py

```diff
@@ -236,15 +229,15 @@ def create_input_json(
             )
 
     # check if all required files are available
-    extra = list(set(list_assets) ^ set(ALLOWED_FILES))
+    extra = list(set(list_assets) ^ set(REQUIRED_FILES))
 
     for i in extra:
-        if i in required_files_list:
+        if i in REQUIRED_FILES:
             logging.error(
                 "Required input file %s is missing! Please add it"
                 "into %s." % (i, os.path.join(input_directory))
             )
-        elif i in ALLOWED_FILES:
+        elif i in REQUIRED_FILES:
             logging.debug(
                 "No %s" % i + ".csv file found. This is an " "accepted option."
             )
```

Comment:
```suggestion
```
This should be deleted - as it is NOT accepted to not have one of the required files.

Reply:
What could happen, though, is that a file that is in "csv_elements" is not in REQUIRED_FILES, as it is a storage_*.csv. This is accepted, and should produce a warning.